distribution/docs/src/main/resources/content/_managing/_configuring/web-context-policy-manager.adoc

```diff
@@ -47,6 +52,20 @@ Any web context that allows or requires specific authentication types should be
 
 |===
 
+==== Terminating Authentication Types
+Terminating authentication types are authentication types where, once triggered, must either obtain credentials or forbid access to the system.
+No succeeding configured authentication types will be used after a terminating authentication type is triggered.
+The terminating auth types are `BASIC`, `SAML`, and `OIDC`.
+
+For example: if a context is protected by the `BASIC` authorization type, the user must enter in proper basic credentials in order to access the system.
+Entering invalid credentials will continuously re-prompt for valid credentials.
+Attempting to bypass this basic prompt will result in a 403 forbidden response, even if guest access is allowed or if a succeeding authorization type is specified on the context.
+
+Contrarily, the `PKI` authentication type is non-terminating.
+If provided an invalid certificate, or if bypassed, the PKI authentication type will fail out and continue to the next specified authentication type for the context path.
```

Comment:
I'm not sure this is true. If the cert is invalid, then we should still fail. If they don't attempt 2-way TLS then we would continue on to whatever comes after PKI